### PR TITLE
Changing expenditure_date to date

### PR DESCRIPTION
**Pull request by LinneaHarts**

### CODE REVIEW

**Comment by LinneaHarts** [comment]
Any comments on this?

File: SD_DB_Setup.sql [hunk 1]
```diff
@@ -1,8 +1,11 @@
-drop database SisterDistrict_dev;
+#drop database SisterDistrict_dev;
+drop database SisterDistrict;
```
Comment (jjshanks) [inline SD_DB_Setup.sql]: can you add IF EXISTS to this, otherwise the script will fail on an existing db

**Comment by jjshanks** [comment]
:shipit:

File: SD_DB_Setup.sql [hunk 2]
```diff
@@ -148,7 +151,7 @@ create table race_expenditures
     amount float,
     date_modified timestamp not null default current_timestamp(),
     user_modified varchar(20) not null,
-    expenditure_date timestamp,
+    expenditure_date date,
```
Comment (nsudarsanam) [inline SD_DB_Setup.sql:155]: looks good

**Review by nsudarsanam — APPROVED** [review]

**Pull request merged**